packages/inertia-react/src/App.js

```diff
@@ -8,6 +8,7 @@ export default function App({
   children,
   initialPage,
   resolveComponent,
+  resolveErrors,
```

Comment:
```suggestion
  resolveErrors = resolveErrors,
```

Will also need to add this method definition about, like `transformProps`.

```js
const resolveErrors = page => (page.props.errors || {})
```

I'd also love to rename the `transformProps` callback from `identity` to `transformProps`. I really don't understand the name `identity` at all, and now we have two callbacks, so we really need better naming. Maybe @sebastiandedeyne can provide some insight here.